src/mmw/js/src/user/templates/baseModal.html

```diff
@@ -16,6 +16,15 @@
     </div>
 {% endmacro %}
 
+{% macro hidden_checkbox(value, name, label='') %}
+    <div class="checkbox" style="display: none;">
+        <label>
+            <input name="{{ name }}" type="checkbox" id="{{ name }}" checked>
```

Comment:
According to the card (emphasis added)
> Users signing up should **not** have this field marked as accepted

I think we probably will also want to clear out that dummy text as well.